ascii.py

```diff
@@ -0,0 +1,31 @@
+class Ascii():
```

Comment:
Two thoughts here:

- `Ascii` is not, by itself, a great name, since it doesn't describe what this thing does (draw a game board). (If you had a set of different ways to draw the board and this was the ASCII variant of that, it would make sense.)
- This is a class, but it has no internal state and exists only to call one method on it, where the method receives all the information it needs. Why have this be a class?

Reply:
Agreed. Will refactor and rename.